### ISSUE TITLE
Provide cockpit/ws image for arm64

### ISSUE THREAD

**Issue by jnohlgard** [cockpit-project/cockpit#17569]
Hi,
I am using Fedora IoT on 64 bit ARM (aarch64). The cockpit packages in dnf are available on aarch64, but it would be nice if the container images also were built for this. Currently x86_64 is the only architecture available for cockpit-ws at Docker Hub.

Docker hub has support for multiple architectures in the same repo, see https://www.docker.com/blog/multi-arch-build-and-images-the-simple-way/
Example of a multi arch repo: https://hub.docker.com/_/influxdb?tab=tags&page=1&ordering=last_updated

**Comment by realk1ko** [comment]
@martinpitt I've noticed the [recent changes in the workflows](https://github.com/cockpit-project/cockpit/commit/ad76a24c0532e2e137cf125adec7520a136f17d9) for the quay.io cockpit/ws container. Are there any plans on making this a "proper" alternative installation approach or is the container simply used for testing purposes?

**Comment by martinpitt** [comment]
No, that container is an official product which we support. [Github does not have ARM runners](https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#jobsjob_idruns-on). quay.io container builds are back -- but not sure if they support ARM somehow, I don't find any architecture selection in the build trigger config.

**Comment by realk1ko** [comment]
I've just opened #17935 that _should_ solve this issue. There are a few open topics I added to the PR, hence the draft status.

I'd appreciate any guidance on some of the topics there, especially about the Docker/podman compatibility that might not be necessary. I'm pretty sure the solution isn't the best as of now, but it'll get there. 🙂
